src/server/api/parameters_parser.cpp

```diff
@@ -55,6 +55,11 @@ boost::optional<ParameterT> parseParameters(std::string::iterator &iter,
         // iterator to the failing position. Extract the position from the exception ourselves.
         iter = failure.first;
     }
+    catch (const boost::numeric::bad_numeric_cast &e)
+    {
+        // this can happen if we get bad numeric values in the request, just handle
+        // as normal parser error
+    }
```

Comment:
How does this make sure invalid coordinates making the parser fail? Relying on the Coordinate type's boost numeric_cast calls in the ctor? Shouldn't we rely on a coordinate's `.IsValid()` member function instead?